pubspec.lock

```diff
@@ -762,7 +776,7 @@ packages:
       name: provider
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "5.0.0"
+    version: "6.0.3"
```

Comment:
same as the above for what i mentioned, jumping from `5.0.0` to `6.0.3` for the package `platform` as well as for `path_provider`.